src/API/files/files.service.test.js

```diff
@@ -32,7 +33,7 @@ describe('files service', () => {
         });
 
         it('should clean files folder content', () => {
-            del.should.calledWith(`${config.filesPath}/**`);
+            del.should.calledWith(path.join(config.filesPath,'/**'));
```

Comment:
I think we should remove '/' from '/**'.
-- path.join(config.filesPath,'**')
!fix